packages/website/src/SideNavigation.tsx

```diff
@@ -90,6 +90,7 @@ const CurrentNavigation = () => {
             <CollapsibleSideSection title="Layout">
                 <NavLink href="/layout/Header" label="Header" />
                 <NavLink href="/layout/Table" label="Table" />
+                <NavLink href="/layout/ModalWizard" label="ModalWizard" />
```

Comment:
Lets keep the alphabetical order going 🤓 

```suggestion
                <NavLink href="/layout/Header" label="Header" />
                <NavLink href="/layout/ModalWizard" label="ModalWizard" />
                <NavLink href="/layout/Table" label="Table" />
```